cmd/token/token.go

```diff
@@ -26,9 +28,10 @@ func NewTokenCmd(f *factory.Factory) *cobra.Command {
 	}
 
 	var tokenCmd = &cobra.Command{
-		Use:   "token",
-		Short: "Perform actions on Admin, Claims and Entitlement tokens",
-		Long:  `The token command allows you to renew or revoke tokens used in the Collective.`,
+		Use:     "token",
+		Aliases: []string{"tokens"},
```

Comment:
Let's keep it consistent with all the other commands, and not put plural on our noun commands.